.goreleaser-next.yml

```diff
@@ -0,0 +1,153 @@
+---
+project_name: epinio-ui
+
+archives:
+  - name_template: '{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    replacements:
+      amd64: x86_64
+    format: binary
+    format_overrides:
+      - goos: windows
+        format: zip
+
+before:
+  hooks:
+    - sh -c "cd src/jetstream && go mod download"
+
+# binary builds
+builds:
+  
+  # amd64
+  - id: epinio-ui-amd64
+    dir: src/jetstream
+    binary: epinio-ui
+    ldflags:
+      - -w -s -X "main.appVersion={{ .Version }}"
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CC=gcc
```

Comment:
i would recommend to change to
```
      - CC=x86_64-linux-gnu-gcc
      - CXX=x86_64-linux-gnu-g++
```

in this case it will work if someone going to try to build it on arm host machine (M1 Mac for example)

Reply:
Ah, nice! Thanks